npm/postinstall.js

```diff
@@ -2,12 +2,30 @@ const stat = require("fs").stat
 const spawn = require("child_process").spawn
 const join = require("path").join
 const pkg = require("../package.json")
-
+const fs = require("fs-extra")
```

Comment:
Why ? fs-promise is just a wrapper of fs-extra. I'm using async operation here :). And I don't like a mix of callback and promise.